Dockerfile-linux

```diff
@@ -30,11 +30,11 @@ RUN \
               --disable-shared \
               --enable-static \
               --with-pic && \
-  make && make check && make install
+  make && make install
```

Comment:
Why is `make check` removed?

Reply:
Removed it for testing since it takes about 30 mins. Putting it back in soon.